integrated_channels/integrated_channel/management/commands/transmit_learner_data.py

```diff
@@ -23,6 +23,7 @@ class Command(IntegratedChannelCommandMixin, BaseCommand):
     help = _('''
     Transmit Enterprise learner course completion data for the given EnterpriseCustomer.
     ''')
+    stealth_options = ('enterprise_customer_slug', 'user1', 'user2')
```

Comment:
Why was this needed?

Reply:
Before **Django2.0**, any arguments passed to the `call_command()` function are only evaluated on time of their usage in the function but after **Django2.0**, the arguments are validated during the function call. We can either add the arguments by `argparser` or can add in `stealth_options` as I did in my change. See [Django2.0 release_notes](https://docs.djangoproject.com/en/2.0/releases/2.0/#call-command-validates-the-options-it-receives) for the details.